smoke_tests/request_and_confirm_test.rb

```diff
@@ -0,0 +1,132 @@
+require 'capybara/poltergeist'
+
+class SmokeTest
+  include Capybara::DSL
+
+  PRISON_EMAIL = 'pvb.cardiff@maildrop.dsd.io'
+  VISITOR_EMAIL = 'test@digital.justice.gov.uk'
+
+  def run
+    with_custom_config
+    run_internal
+  rescue StandardError => e
+    pp e
+    pp e.backtrace
+    save_screenshot('error.png', full: true)
+  end
+
+  private
+  def run_internal
+    visit '/prisoner'
+    enter_prisoner_information
+    continue
+    
+    raise "Can't find Visitor 1" unless page.has_content?('Visitor 1')
```

Comment:
Can we get more info from this (for example, some detail about what the page actually said, or what the last response was or something)? At the moment, the smoke test just fails with this:

```
#<RuntimeError: Can't find Visitor 1>
["/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/smoke_tests/request_and_confirm_test.rb:24:in `run_internal'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/smoke_tests/request_and_confirm_test.rb:11:in `run'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/lib/tasks/smoke.rake:5:in `block (2 levels) in <top (required)>'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/task.rb:240:in `call'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/task.rb:240:in `block in execute'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/task.rb:235:in `each'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/task.rb:235:in `execute'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/task.rb:179:in `block in invoke_with_call_chain'",
 "/usr/local/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/task.rb:172:in `invoke_with_call_chain'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/task.rb:165:in `invoke'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/application.rb:150:in `invoke_task'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/application.rb:106:in `block (2 levels) in top_level'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/application.rb:106:in `each'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/application.rb:106:in `block in top_level'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/application.rb:115:in `run_with_threads'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/application.rb:100:in `top_level'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/application.rb:78:in `block in run'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/application.rb:176:in `standard_exception_handling'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/lib/rake/application.rb:75:in `run'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/gems/rake-10.3.2/bin/rake:33:in `<top (required)>'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/bin/rake:23:in `load'",
 "/srv/jenkins/jobs/smoketest-production/workspace/prison-visits/vendor/ruby/2.0.0/bin/rake:23:in `<main>'"]
```